src/widgets/helper/ChannelView.hpp

```diff
@@ -75,7 +76,23 @@ class ChannelView final : public BaseWidget
         Search,
     };
 
-    explicit ChannelView(BaseWidget *parent = nullptr, Split *split = nullptr,
+    /// Creates a channel view without a split.
+    /// In such a channel, usercards and reply-threads can't be opened.
+    ///
+    /// @param parent The parent of this widget. Can be `nullptr`.
+    /// @param context The context in which this view is shown (e.g. as a usercard).
+    /// @param messagesLimit The maximum amount of messages this view will display.
+    explicit ChannelView(QWidget *parent, Context context = Context::None,
+                         size_t messagesLimit = 1000);
+
+    /// Creates a channel view in a split.
+    ///
+    /// @param parent The parent of this widget.
+    /// @param split The split contianing this widget.
+    ///              @a split must be in the widget tree of @a parent.
```

Comment:
what are the `@`'s for here? some docstring magic?

Reply:
It emphasizes the word ([docs](https://www.doxygen.nl/manual/commands.html#cmda); clangd doesn't display that right now).